invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/DeleteButton.js

```diff
@@ -74,18 +99,94 @@ export const DeleteButton = (props) => {
         onClose={handleClose}
         size="tiny"
       >
-        <Modal.Content>{confirmationMessage}</Modal.Content>
+        <Modal.Header>{i18next.t("Permanently delete community")}</Modal.Header>
+        <Modal.Content>
+          <Trans>
+            Are you <b>absolutely sure</b> you want to delete the community?
+          </Trans>
+        </Modal.Content>
+        <Modal.Content>
+          <Message negative>
+            <Message.Header>
+              <Grid columns={2} verticalAlign="middle">
+                <Grid.Column width={1}>
+                  <Icon name="warning sign" />
+                </Grid.Column>
+                <Grid.Column width={15}>
+                  {i18next.t("This action CANNOT be undone!")}
+                </Grid.Column>
+              </Grid>
+            </Message.Header>
+            <Message.Content>
+              <Checkbox
+                label={
+                  /* eslint-disable-next-line jsx-a11y/label-has-associated-control */
+                  <label>
+                    <Trans>
+                      <b>All the members</b> will be removed from the community.
```

Comment:
Minor: It's recommended [in the HTML5 specs](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-b-element) to use more semantic tags such as `<em>` or `<strong>` over `<b>` tags if applicable.